source/docs/advanced/performance/preloading-routes.md

```diff
@@ -0,0 +1,183 @@
+
+This is why if you have trouble preloading data of one of your custom routes, you first need to make sure that the `preload` property is available. If it is not, please refer to [the first part of this documentation](#How-to-preload-data).
+
+Additionally, it will also preload the data needed for the layouts around the matched route. This means that you can also set a `preload` static property on layouts and inner layouts.
+
+## How to preload routes with parameters?
+
+You can now preload static pages such as `/contact` or `/home`. But what about `/product/:sku` or `/my-seo-friendly-product-url`?
+
+The `usePreload` hook will take care of this for you by calling the `preload` static property of your component with an object containing:
+
+* `match`: a [React Router object](https://reacttraining.com/react-router/web/api/match) that describes the currently matched route.
+    * `params` (object): Key/value pairs parsed from the URL corresponding to the dynamic segments of the path
+    * `isExact` (boolean): true if the entire URL was matched (no trailing characters)
+    * `path` (string): The path pattern used to match. Useful for building nested <Route>s
+    * `url` (string): The matched portion of the URL. Useful for building nested <Link>s
+* `location`: a [React Router object](https://reacttraining.com/react-router/web/api/location) that describes the url that you passed to the `preload` function (`/venia-dresses` in the example above)
+
+These two parameters allow you to almost expect to have the same properties between the `options` and `preloadOptions` functions of `graphqlWithPreload`.
+
+For instance, let's consider that the URL `/product/:sku` should display the following component.
+
+```js
+import React from "react";
+import graphqlWithPreload from "web/core/apollo/graphqlWithPreload";
+
+const Product = props => {
+  return <div>{props.product.name}</div>;
+};
+
+const EnhancedProduct = (
+  graphqlWithPreload(ProductQuery, {
+    options: props => ({
+      variables: {
+        sku: props.match.params.sku
+      }
+    }),
+    preloadOptions: params => ({
+      variables: {
+        sku: params.match.params.sku
+      }
+    })
+  })
+)(Product);
+```
+
+Then this means that if you call `preload("/product/VD12")`, in `params.match.params.sku` (line 17) you will have the value `VD12`.
+
+If you need search parameters in the URL (e.g. `?param=value`), it will be available at `params.location.search`.
+
+## Front-Commerce Fast Mode
+
+To conclude, if you want your application to feel even faster to your users, you need to:
+* add a `preload` static property to your routes thanks to `graphqlWithPreload`
+* use `usePreload` hook through the default Link component in Front-Commerce or manually in your own events.
+
+This also enables a wider range of optimizations such as Front-Commerce Fast Mode. Please refer to [FRONT_COMMERCE_FAST] for more information.
```

Comment:
The `FRONT_COMMERCE_FAST` link is missing here.

Reply:
```suggestion
This also enables a wider range of optimizations such as Front-Commerce Fast Mode. Please refer to [FRONT_COMMERCE_FAST](/docs/advanced/performance/faster-server-side-rendering.html) for more information.
```